File: C++/Data-Structures/STACKS/stacks.cpp
#include "Stack.hpp"
using namespace std;

int main()
{
    Stack<int> s;
    s.push(1);
    s.push(2);
    s.push(3);
    s.push(4);
    s.push(5);
    s.pop();
    s.pop();
    //s.size();
    //s.print();
    // cout << s.empty();
    // Stack<string> str;
    // str.push("hello");
    // str.push("world");
    // str.print();
    Stack<int> s1;
    s.copy(s1);
    s1.print();
    //cout << s1.size();
    return 0;
}

File: Java/Data-Structures/STACKS/INBUILT-STACK/Stacks.java
import java.util.Stack;
public class Stacks
{
    public static void main(String[] args)
    {
        Stack<Integer> S = new Stack<Integer>();
        S.push(1);
        S.push(2);
        S.push(3);
        S.pop();
        S.push(4);
        
        System.out.println(S);
        System.out.println("size: "+S.size());
        System.out.println("empty: "+S.empty());
        System.out.println("peek: "+S.peek());
        System.out.println("search: "+S.search(1));
    }
}